code/__HELPERS/time.dm

```diff
@@ -95,7 +95,7 @@ proc/isDay(var/month, var/day)
 /proc/seconds_to_time(var/seconds as num)
```

Comment:
Thankfully this proc is only used in `brigdoors.dm` so you dont have to go around and update it everywhere.